tests/test_records.py

```diff
@@ -1,6 +1,10 @@
 import pytest
 
-from installer.records import SuperfluousRecordColumnsWarning, parse_record_file
+from installer.records import (
+    RecordItem,
```

Comment:
Looks like we should rebase this PR on #12 (or wait for that to merge, then rebase on master).

This would also mean changing the tests to be a method on the relevant class.